doc/source/Getting_started/Troubleshooting.rst

```diff
@@ -7,10 +7,13 @@ Installation
 
 Error installing Python or Conda
 --------------------------------
-It may happen, that IT Department doesn't allow users to install a Python interpreter in the machine.
-In this case user can use the interpreter available in installation path of Ansys Electronics Desktop.
-Please note that Python 3.7 is available in AEDT <= 2023R1 while Python 3.10 is available on Ansys Electronics
-Desktop 2023 R2.
+Sometimes users are not allowed to install a Python interpreter.
+In this case, a Python interpreter is available in installation path of
+Ansys Electronics Desktop.
+Note that Python 3.7 is available in AEDT <= 2023R1 while Ansys Electronics Desktop 2023 R2
+is shipped with Python 3.10.
+
+*Python 3.7 interpreter path for the 2023R1 Installation:*
```

Comment:
```suggestion
In this case, you can use the Python interpreter available in the AEDT installation.

.. note::
Python 3.7 is available in AEDT 2023 R1 and earlier. Python 3.10 is available in ASEDT 2023 R2.

Here is the path to the Python 3.7 interpreter for the 2023 R1 installation:
```
No need for italics. An Ansys release is always in a format like 2023 R1 (with a space) for public-facing documentation.